spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingServiceProperties.java

```diff
@@ -235,6 +236,13 @@ public String getBinder(String channelName) {
 			dataBinder.setIgnoreUnknownFields(ignoreUnknownProperties);
 			dataBinder.setDisallowedFields(bindingPropertyFields);
 			dataBinder.bind(new PropertySourcesPropertyValues(applicationContext.getEnvironment().getPropertySources()));
+			CustomValidatorBean customValidator = new CustomValidatorBean();
```

Comment:
This has been rendered obsolete, as we don't do our own binding anymore. However, it would be useful to have an integration tests where invalid values are actually detected.